user_guide_src/source/changelogs/v4.4.0.rst

```diff
@@ -52,15 +52,24 @@ Parameter Type Changes
 
 - **Services:** The first parameter of ``Services::security()`` has been
   changed from ``Config\App`` to ``Config\Security``.
-- **Session:** The second parameter of ``Session::__construct()`` has been
-  changed from ``Config\App`` to ``Config\Session``.
-- **Session:** The first parameter of ``__construct()`` in ``BaseHandler``,
-  ``DatabaseHandler``, ``FileHandler``, ``MemcachedHandler``, and ``RedisHandler``
-  has been changed from ``Config\App`` to ``Config\Session``.
+- **Session:**
+    - The second parameter of ``Session::__construct()`` has been changed from
+      ``Config\App`` to ``Config\Session``.
+    - The first parameter of ``__construct()`` in ``BaseHandler``,
```

Comment:
I realize this was just a formatting change but could you please specify this more?
```suggestion
    - The first parameter of ``__construct()`` in the database's ``BaseHandler``,
```